src/Microsoft.DotNet.Interactive/KernelSupportsNugetExtensions.cs

```diff
@@ -178,36 +166,58 @@ public static string GetDisplayValueId(PackageReference package)
             public static IEqualityComparer<PackageReference> Instance { get; } = new PackageReferenceComparer();
         }
 
+        private static void CreateOrUpdateDisplayValue(KernelInvocationContext context, string name, object content, string mimeType)
```

Comment:
Why 'object`?

Reply:
So that we can use string or IHTMLContent.  I use a string to empty the message out.

Reply:
I suppose I could have used null.

Reply:
`null` which actually get written out as the string "null" by the default formatter.